pkg/expr/nodes.go

```diff
@@ -103,6 +103,10 @@ func (gn *CMDNode) NeedsVars() []string {
 // other nodes they must have already been executed and their results must
 // already by in vars.
 func (gn *CMDNode) Execute(ctx context.Context, now time.Time, vars mathexp.Vars, s *Service) (mathexp.Results, error) {
+	// If this is a SQL command, pass the cell limit from service config
+	if sqlCmd, ok := gn.Command.(*SQLCommand); ok {
+		sqlCmd.limit = s.cfg.SQLExpressionCellLimit
```

Comment:
i'm a bit unhappy that we are mutating the sql-command object here.
i would much prefer the `s.cfg.SQLExpressionCellLimit` provided into `NewSQLCommand` as a parameter. that way the sql-expression object is not changing ever, the limit is hardcoded in it.

though i undrstand that at this point we have access to the needed settings-object, so it's a pragmatic choice to do it here.

also there may be some other reason i'm not familiar with (i'm not familiar much with the expression-nodes-system)

still, if possible, sending it into `NewSQLCommand` would be great :)

Reply:
Ok, I've made this change, please see what you think. The code change was a tiny bit more involved, and the test a little longer, but I wouldn't be surprised if you think this is a good change altogether. 

Also I have a question - should we prevent all mutation of these built nodes once they have been built? Definitely not one for this PR, and maybe more of a Golang question than an SSE question, but basically wondering if this can be done, and if people do it in practice?